app/uk/gov/hmrc/agentclientrelationships/model/InActiveRelationship.scala

```diff
@@ -43,13 +42,13 @@ object InactiveRelationship {
       val clientType =
         if ((json \ "individual").asOpt[JsValue].isDefined) "personal" else "business"
       val service = clientId match {
-        case _ if clientId.matches(CgtRef.cgtRegex)                => HMRC_CGT_PD
-        case _ if PptRef.isValid(clientId)                         => HMRC_PPT_ORG
-        case _ if Vrn.isValid(clientId)                            => HMRC_MTD_VAT
-        case _ if Utr.isValid(clientId)                            => HMRC_TERS_ORG
-        case _ if clientType == "business" & Urn.isValid(clientId) => HMRC_TERSNT_ORG
-        case _ if MtdItId.isValid(clientId)                        => HMRC_MTD_IT
-        case _ if CbcId.isValid(clientId)                          => HMRC_CBC_ORG // doesn't matter whether uk or non uk apparently
+        case _ if clientId.matches(CgtRef.cgtRegex)                => "HMRC-CGT-PD"
+        case _ if PptRef.isValid(clientId)                         => "HMRC-PPT-ORG"
+        case _ if Vrn.isValid(clientId)                            => "HMRC-MTD-VAT"
+        case _ if Utr.isValid(clientId)                            => "HMRC-TERS-ORG"
+        case _ if clientType == "business" & Urn.isValid(clientId) => "HMRC-TERSNT-ORG"
+        case _ if MtdItId.isValid(clientId)                        => "HMRC-MTD-IT"
+        case _ if CbcId.isValid(clientId)                          => "HMRC-CBC-ORG"
```

Comment:
have you checked that a CBC tax id won't match any of the regex's above it?

Reply:
That's a good point, I'll move that case nearer the top of the list. The CBC id is quite distinctive so it should be safer to check it first